codecs/resize/pkg/resize_bg.d.ts

```diff
@@ -3,4 +3,4 @@ export const memory: WebAssembly.Memory;
 export function __wbindgen_global_argument_ptr(): number;
 export function __wbindgen_malloc(a: number): number;
 export function __wbindgen_free(a: number, b: number): void;
-export function resize(a: number, b: number, c: number, d: number, e: number, f: number, g: number, h: number): void;
+export function resize(a: number, b: number, c: number, d: number, e: number, f: number, g: number, h: number, i: number): void;
```

Comment:
Any idea why the bool is coming out as 'number'?

Reply:
For the `resize_bg.d.ts` file that’s actually correct. Wasm doesn’t have a `bool` type. `resize.d.ts`, the types for the wrapper functions, has `boolean`.